lib/net.c

```diff
@@ -104,6 +104,11 @@ static const struct rte_eth_conf gatekeeper_template_port_conf = {
 		 * the static config as the field mtu and is set
 		 * in init_port(). See the documentation of member
 		 * mtu of struct gatekeeper_if for more information.
+		 *
+		 * XXX Remove last hardware requirements for
+		 * Gatekeeper. Make jumbo frame offloading
+		 * configurable and add software alternatives for
+		 * checksums, mostly for the virtio environment.
```

Comment:
Instead of adding this `XXX` comment, open an issue on GitHub. It makes things easier to track.